src/sql/data.d.ts

```diff
@@ -1410,6 +1410,38 @@ declare module 'data' {
 		postMessage(message: any): Thenable<any>;
 	}
 
+	export interface WebviewWidget {
```

Comment:
Assuming we're going to want other widgets that aren't web views, I wonder if there's a better way to organize this instead of leaving it at the top-level inside data. Maybe under a "widgets" namespace or something like that?